lib/Runtime/Types/RecyclableObject.h

```diff
@@ -278,8 +278,8 @@ namespace Js {
         virtual BOOL GetEnumerator(BOOL enumNonEnumerable, Var* enumerator, ScriptContext * requestContext, bool preferSnapshotSemantics = true, bool enumSymbols = false);
         virtual BOOL ToPrimitive(JavascriptHint hint, Var* value, ScriptContext * requestContext);
         virtual BOOL SetAccessors(PropertyId propertyId, Var getter, Var setter, PropertyOperationFlags flags = PropertyOperation_None);
-        virtual BOOL Equals(Var other, BOOL* value, ScriptContext* requestContext);
-        virtual BOOL StrictEquals(Var other, BOOL* value, ScriptContext* requestContext);
+        virtual BOOL Equals(__in Var other, __out BOOL* value, ScriptContext* requestContext);
+        virtual BOOL StrictEquals(__in Var other, __out BOOL* value, ScriptContext* requestContext);
```

Comment:
Code analysis shows C6101 warning if out parameter is not set in some paths

---
In reply to: [73197400](https://github.com/Microsoft/ChakraCore/pull/1365#discussion_r73197400) [](ancestors = 73197400)